cypress/integration/loginWithRedirect.js

```diff
@@ -47,4 +48,37 @@ describe('loginWithRedirect', function() {
       shouldBe(pageParams.scope, 'openid profile email test test2');
     });
   });
+
+  it('can peform the login flow', () => {
+    whenReady().then(win => {
+      cy.loginNoCallback();
+
+      cy.url().then(url => shouldInclude(url, 'https://brucke.auth0.com'));
+
+      cy.get('#loaded').then(() => {
+        expect(win.sessionStorage.getItem('a0.spajs.txs')).to.exist;
+
+        cy.handleRedirectCallback().then(() => {
+          expect(win.sessionStorage.getItem('a0.spajs.txs')).to.not.exist;
+        });
+      });
+    });
+  });
+
+  it('can perform the login flow with cookie transactions', () => {
+    whenReady().then(win => {
+      cy.toggleSwitch('cookie-txns');
+      cy.loginNoCallback();
+
+      cy.url().then(url => shouldInclude(url, 'https://brucke.auth0.com'));
+
+      cy.get('#loaded').then(() => {
+        cy.getCookie('a0.spajs.txs').should('exist');
+
+        cy.handleRedirectCallback().then(() => {
+          cy.getCookie('a0.spajs.txs').should('not.exist');
+        });
+      });
+    });
```

Comment:
Should we test the expiration time being set or is that not so useful? Just to ensure everything is set up correctly to clean up the cookie when `handleRedirectCallback` was never called.

Reply:
I don't see why not. I'll do that in a different test though as this one is checking for the _non_-existance of that cookie.

Reply:
Updated!